Apps/FCM/ios/NotificationServiceExtension/Info.plist

```diff
@@ -7,7 +7,7 @@
 		<key>NSExtensionPointIdentifier</key>
 		<string>com.apple.usernotifications.service</string>
 		<key>NSExtensionPrincipalClass</key>
-		<string>$(PRODUCT_MODULE_NAME).NotificationService</string>
```

Comment:
Does it happen by default? did any change introduce it? do we know how did it get here?

Reply:
I did check git for the change but did not find much, seems to be there by default. It brings up a question that how was it working earlier, but I am not sure of the answer ! It could be an Xcode thing, Xcode 15 shows clear message than former version that helped me figure out the issue.